src/components/Calculator.js

```diff
@@ -1,27 +1,21 @@
 /* eslint-disable react/prefer-stateless-function */
```

Comment:
- [ ] This linter disabling is unnecessary. ESLint does not return any errors without it and your code works just fine. Kindly remove it.